### DISCUSSION TITLE
Paths in Vignettes Not Being Created/ Found

### DISCUSSION THREAD

**Discussion by anthonyjlau** [sailuh/kaiaulu#272] [Q&A] [answered]
I have noticed that a reoccurring error I have when trying to run the vignette notebooks is that files or paths cannot be found. 

An example of this is from the github_api_showcase.Rmd when I run:
https://github.com/sailuh/kaiaulu/blob/9294a9bb92506b76424b0fcc7c07f6b74d5ce53d/vignettes/github_api_showcase.Rmd#L91-L92
I get an error stating that it cannot create a directory called *commit* and it cannot find a *sailuh_kaiaulu_commit_p_1.json* file. 

Another example is when I run line_metrics_showcase.Rmd:
https://github.com/sailuh/kaiaulu/blob/9294a9bb92506b76424b0fcc7c07f6b74d5ce53d/vignettes/line_metrics_showcase.Rmd#L81
I get an error very similar to the one in #147 . I looked through that discussion but I could not find, or maybe not understand, a solution.

I have looked through the apr.yml and kaiaulu.yml config files to see the paths that are assigned to the variables. 

I was wondering if I could get some advice on how to setup my config files, the tools.yml file, or my directory setup. Thanks!

**Comment by carlosparadis** [comment]
You have to specify on the project configuration file the path you expect the files to be downloaded. Could you check the folder where `commit` is trying to be created exists?

**Reply by anthonyjlau** [reply, marked as answer]
I found that the paths had to be manually created and the project configuration file had to be changed from the default. I got rid of the default * .. * notation in the paths and just used a direct path to the folder.
